config.yml

```diff
@@ -7723,3 +7441,355 @@ production_recipes:
+        amount: 512
+      Netherwart:
+        material: NETHER_STALK 
+        amount: 640
+      Sand:
+        material: SAND
+        amount: 32
+    outputs:
+      Wood Crate:
+        material: STORAGE_MINECART
+        amount: 1
+        display_name: Wood Crate
+        lore: Packaged local market goods. Useless for resource extraction without further refinement.
+  Wood_XP_Recipe_1:
+    name: Craft XP Bottles 1
+    production_time: 30
+    inputs:
+      Hay Bale:
+        material: HAY_BLOCK
+        amount: 896
+      Glass Bottle:
+        material: GLASS_BOTTLE
+        amount: 8
+    outputs:
+      Exp Bottle:
+        materials: EXP_BOTTLE
+        amount: 8
+  Wood_XP_Recipe_2:
+    name: Craft XP Bottles 2
+    production_time: 30
+    inputs:
+      Raw Potato:
+        material: POTATO
+        amount: 1280
+      Glass Bottle:
+        material: GLASS_BOTTLE
+        amount: 4
+    outputs:
+      Exp Bottle:
+        materials: EXP_BOTTLE
+        amount: 4
+  Wood_XP_Recipe_3:
+    name: Craft XP Bottles 3
+    production_time: 30
+    inputs:
+      Cocoa:
+        material: INK SACK
+        durability: 3
+        amount: 640
+      Glass Bottle:
+        material: GLASS_BOTTLE
+        amount: 4
+    outputs:
+      Exp Bottle:
+        materials: EXP_BOTTLE
+        amount: 4
+  Wood_XP_Recipe_4:
+    name: Craft XP Bottles 4
+    production_time: 30
+    inputs:
+      Nether Wart:
+        material: NETHER_STALK
+        amount: 1280
+      Glass Bottle:
+        material: GLASS_BOTTLE
+        amount: 6
+    outputs:
+      Exp Bottle:
+        materials: EXP_BOTTLE
+        amount: 6
+  Iron_Crate:
+    name: Package Iron Crate
+    production_time: 40
+    inputs:
+      Wood Crate:
+        material: STORAGE_MINECART
+        amount: 4
+        display_name: Wood Crate
+        lore: Packaged local market goods. Useless for resource extraction without further refinement.
+      Vine:
+        material: VINE
+        amount: 640
+      Clay Blocks:
+        material: CLAY
+        amount: 64
+      Cake:
+        material: CAKE
+        amount: 16
+      Red Mushroom:
+        material: RED_MUSHROOM
+        amount: 64
+      Apple:
+        material: APPLE
+        amount: 64
+      Netherrack:
+        material: NETHERRACK
+        amount: 96
+      Flint:
+        material: FLINT
+        amount: 64
+    outputs:
+      Iron Crate:
+        material: POWERED_MINECART
+        amount: 1
+        display_name: Iron Crate
+        lore: Processed market supplies. It can be converted into XP but it's not very efficient...
+  Iron_XP_Recipe_1:
+    name: Craft XP Bottles 1
+    production_time: 10
+    inputs:
+      Bread:
+        material: BREAD
+        amount: 640
+      Netherwart:
+        material: NETHER_STALK 
+        amount: 320
+      Baked Potato:
+        material: BAKED_POTATO
+        amount: 640
+      Glass Bottle:
+        material: GLASS_BOTTLE
+        durability: 16

```

Comment:
My dumb ass forgetting to remove durability and trying to edit the amount.

I'll get that fixed up tonight. Should I be planning to squash or no squash still?